### ISSUE TITLE
[cetic/nifi] pod stops only after terminationGracePeriodSeconds exceeded

### ISSUE THREAD

**Issue by gforeman02** [cetic/helm-nifi#239]
**Describe the bug**
When a pod in a cluster is triggered to stop, the offload node process can take some time to complete.  Increasing the terminationGracePeriodSeconds to 600 provides enough time for the server container to complete offloading with a generous buffer (in my environment).  While the Nifi server container does stop, the sidecar containers continue to run.  The result is the pod does not terminate until the terminationGracePeriodSeconds passes.

**Version of Helm, Kubernetes and the Nifi chart**:
Helm: 3.5.1
Kubernetes: 1.20.5
Nifi chart: 1.0.5

**What happened**:
See description.

**What you expected to happen**:
The termination of the pod should complete when the Nifi server container stops.  The terminationGracePeriodSeconds should be a last resort to terminate the pod.

**How to reproduce it** (as minimally and precisely as possible):
Deploy a Nifi cluster with terminationGracePeriodSeconds to 600.  Reduce the number of nodes in the cluster and note the time.  Monitor the Nifi server container log to ensure offload processes complete.  Monitor how long it takes for the pod to terminate.

**Anything else we need to know**:
None

**Comment by weltonrodrigo** [comment]
I believe this is a duplicate of #158.

This is caused by log tailer sidecars, which for some reason refuse to quit on SIGTERM. I investigated that but could not found a quick solution.

Maybe we need other software besides tail, which respects SIGTERM. tail seems to ignore it when running in the `+F` mode.
